site/app/templates/admin/users/UserForm.twig

```diff
@@ -50,6 +50,10 @@
             Secondary Email:
             <input type="text" name="user_email_secondary" id="user_email_secondary" placeholder="(Optional)" />
         </label>
+        <label for="user_pronouns"> 
+            Pronouns:
+            <input class="readonly" type="text" name="user_pronouns" readonly="readonly" id="user_pronouns" placeholder="(Optional)"/>
```

Comment:
As per our [style guide](https://submitty.org/developer/software_and_system_design/coding_style_guide/css#styling), the `id` and `name` should be `user-pronouns` instead of `user_pronouns`.

Reply:
fixed